02-peer-protocol.md

```diff
@@ -1220,7 +1234,11 @@ protocol, with the following additional caveats.
 
 #### The `tx_add_input` Message
 
-No additional caveats or requirements.
+##### Requirements
+
+The sending node:
+  - if the receiver set `require_confirmed_inputs` in `open_channel2` or `accept_channel2`:
+    - MUST NOT send a `tx_add_input` that contains an unconfirmed input
```

Comment:
what happens if a node receives an input that's unconfirmed? should we add a note on the receiver's side that they should verify received inputs?